tools/topology/topology2/platform/intel/dmic-aec.conf

```diff
@@ -0,0 +1,363 @@
+Object.Dai {
+	DMIC.0 {
+		name			$DMIC0_NAME
+		id 			$DMIC0_ID
+		driver_version		$DMIC_DRIVER_VERSION
+		io_clk			$DMIC_IO_CLK
+		clk_min		500000
+		clk_max		4800000
+		unmute_ramp_time_ms	200
+		# num_pdm_active should always set to 2 but depending on the number of DMIC's
+		# the mic's are enabled or disabled in each PDM.
+		num_pdm_active 	2
+
+		Object.Base.hw_config."DMIC0" {
+			id	0
+		}
+
+		# PDM controller config
+		Object.Base.pdm_config."0" {
+			mic_a_enable	$PDM0_MIC_A_ENABLE
+			mic_b_enable	$PDM0_MIC_B_ENABLE
+			ctrl_id	0
+		}
+		Object.Base.pdm_config."1" {
+			ctrl_id	1
+			mic_a_enable	$PDM1_MIC_A_ENABLE
+			mic_b_enable	$PDM1_MIC_B_ENABLE
+		}
+	}
+}
+
+Object.Dai {
+       DMIC.1 {
+               name                    $DMIC1_NAME
+               id                      $DMIC1_ID
+               dai_index               1
+               driver_version          $DMIC_DRIVER_VERSION
+               io_clk                  $DMIC_IO_CLK
+               sample_rate             16000
+               clk_min         500000
+               clk_max         4800000
+               unmute_ramp_time_ms     200
+               # num_pdm_active should always set to 2 but depending on the number of DMIC's
+               # the mic's are enabled or disabled in each PDM.
+               num_pdm_active  2
+
+               Object.Base.hw_config."DMIC1" {
+                       id      0
+               }
+
+               # PDM controller config
+               Object.Base.pdm_config."3" {
+                       mic_a_enable    $PDM0_MIC_A_ENABLE
+                       mic_b_enable    $PDM0_MIC_B_ENABLE
+                       ctrl_id 0
+               }
+               Object.Base.pdm_config."4" {
+                       ctrl_id 1
+                       mic_a_enable    $PDM1_MIC_A_ENABLE
+                       mic_b_enable    $PDM1_MIC_B_ENABLE
+               }
+       }
+}
```

Comment:
no this makes no  sense to redefine what the DMIC configuration is. The goal is to have a second PCM, not a second configuration of the DMIC.

Reply:
sure, thanks